source/plugins/go/src/oms.go

```diff
@@ -1245,14 +1261,15 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 
 	if len(msgPackEntries) > 0 && ContainerLogsRouteV2 == true {
 		//flush to mdsd
-		if IsAADMSIAuthMode == true && strings.HasPrefix(MdsdContainerLogTagName, MdsdOutputStreamIdTagPrefix) == false {
-			Log("Info::mdsd::obtaining output stream id")
+		if IsAADMSIAuthMode == true && strings.HasPrefix(MdsdContainerLogTagName, MdsdOutputStreamIdTagPrefix) == false && IsGenevaLogsIntegrationEnabled == false {
```

Comment:
This should not be here since we haven't merged Windows AMA changes yet

Reply:
> This should not be here since we haven't merged Windows AMA changes yet

 Removed all the windows ama LA integration code.